src/components/Attachments/AttachmentCarousel/index.tsx

```diff
@@ -102,9 +102,10 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
                 setDownloadButtonVisibility(initialPage !== -1);
             }
 
+            const attachment = targetAttachments.at(initialPage);
```

Comment:
Same, please make sure it doesn't break anything in case `initialPage` is `-1`, cause .at() accepts negative integers while [] not. See https://github.com/Expensify/App/pull/49534#discussion_r1772650621